### PR TITLE
[Clean up] Moving commonly used functions to utility.py

### PR DESCRIPTION
**Pull request by NicEastvillage**
Depends on #12

### CODE REVIEW

**Review by falkecarlsen — CHANGES_REQUESTED** [review]
Looks good but you seem to have removed `display_noisemap()` :upside_down_face:

**Comment by NicEastvillage** [comment]
Don't worry. It is on another branch that I will push in a moment!

File: perlin.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ def get_population_number(edge: sumolib.net.edge.Edge, base: float, centre,
     :return: the scaled noise value as float in [0:1]
     """
     x, y = get_edge_pair_centroid(edge.getShape())
-    return get_perlin_noise(x=float(x), y=float(y), base=base, scale=scale, octaves=octaves) + (
+    return get_perlin_noise(x, y, base=base, scale=scale, octaves=octaves) + (
```
Comment (falkecarlsen) [inline perlin.py:65]: In #12 there is an erratic change casting base to int, but this is not present here - yet this PR depends on #12?

**Review by falkecarlsen — APPROVED** [review]
Looks good, except issue with erratic change as in #12.

File: perlin.py (hunk 1 above)
Comment (NicEastvillage) [reply perlin.py:65]: They are now in sync again!

**Pull request merged**